website/src/pages/api/admin/tasks_availability.ts

```diff
@@ -0,0 +1,24 @@
+import { withRole } from "src/lib/auth";
+
+/**
+ * Returns result of tasks availability query using a dummy user.
+ */
+const handler = withRole("admin", async (req, res) => {
+  const tasksAvailabilityRes = await fetch(`${process.env.FASTAPI_URL}/api/v1/tasks/availability`, {
+    method: "POST",
+    headers: {
+      "X-API-Key": process.env.FASTAPI_KEY,
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
```

Comment:
What's going on with this? are you just polling data for some dummy user? Can we instead use the current admin user?

Reply:
Yes, in issue [#828](https://github.com/LAION-AI/Open-Assistant/issues/828) @andreaskoepf mentioned using a dummy user.
For using the current admin user instead, would you know how to get the current user's user ID?  session.user only contains the display name.

Reply:
Nevermind I think I got it working using the token (I assumed the auth_method is always "local" if that's alright). Now it uses the current admin user.

Reply:
The "problem" with task-availability is that it is user and `lang` tag dependent .. e.g. a user cannot label own messages and  counts as the number of active message-trees are calculated per language (currently the same global setting applies to all languages).

Reply:
Do you mean we should stick with using a dummy user (so that no messages get excluded)?
Also do you think we should add something like a dropdown menu to choose the lang?

Reply:
I still think using a hypothetical dummy user to make the availablity query is fine.

Reply:
Is the prod instance guaranteed to have the dummy user?  If so that'll be safe.

At a later date we should add a backend endpoint that returns the general availability independent of user and language.  But either using the admin or the dummy user is fine

Reply:
Ok, I changed it to use a dummy user. Currently it seems that when calling `/api/v1/tasks_availability` the `PromptRepository` will create the user if they don't exist.

I agree about adding the user independent backend endpoint later.